<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактор тестов</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="teTest"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>929</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>&amp;Тест</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>&amp;Экспорт</string>
     </property>
     <addaction name="actionToText"/>
     <addaction name="actionToPDF"/>
     <addaction name="actionToXML"/>
    </widget>
    <addaction name="actionNewTest"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="menu_2"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>&amp;Вид</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>&amp;Справка</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidgetStruct">
   <property name="windowTitle">
    <string>Структура теста</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContentsStruct">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeWidget" name="twStruct">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetShowTask">
   <property name="windowTitle">
    <string>Задание</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContentsShowTask">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextEdit" name="teTask"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Открыть...</string>
   </property>
   <property name="toolTip">
    <string>Открыть тест</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionToText">
   <property name="text">
    <string>Текст...</string>
   </property>
   <property name="toolTip">
    <string>Экспорт в текстовый файл</string>
   </property>
  </action>
  <action name="actionToPDF">
   <property name="text">
    <string>PDF...</string>
   </property>
   <property name="toolTip">
    <string>Экспорт в PDF</string>
   </property>
  </action>
  <action name="actionToXML">
   <property name="text">
    <string>XML...</string>
   </property>
   <property name="toolTip">
    <string>Экспорт в XML</string>
   </property>
  </action>
  <action name="actionNewTest">
   <property name="text">
    <string>Со&amp;здать...</string>
   </property>
   <property name="toolTip">
    <string>Создать тест</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>В&amp;ыход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;Помощь...</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
